### ISSUE TITLE
Why QNNPACK has better accuracy on MobileNetV2?

### ISSUE THREAD

**Issue by Lyken17** [pytorch/QNNPACK#14]
Hi authors

I notice in `MobilenetV2` section, there is figure comparing the accuracy between QNNPACK 8bit and TFlite 8bit, and your library significantly outperforms TF-Lite. I wonder did you use some special quantize algorithm? Or do a quantized training? 

![](https://code.fb.com/wp-content/uploads/2018/10/QNNPACK7_final3.png)

**Comment by Maratyszcza** [comment]
Caffe2 model for MobileNet v2 is not converted from TensorFlow, it is trained from the scratch, and pre-quantized model is too more accurate than reported in the MobileNet v2 paper.

**Comment by Lyken17** [comment]
Interesting, may I know what techniques you used to boost the accuracy?

**Comment by harouwu** [comment]
Not sure whether we can talk about it now. But stay tuned, we might open source our tool to quantize models and our fp32 model later :)

**Comment by Lyken17** [comment]
ha, understand. Thanks for response. Looking forward to the updates.
